fluentui_drawer/src/main/java/com/microsoft/fluentui/persistentbottomsheet/PersistentBottomSheet.kt

```diff
@@ -95,6 +96,7 @@ class PersistentBottomSheet @JvmOverloads constructor(context: Context, attrs: A
         persistentSheet = findViewInTemplateById(R.id.persistent_bottom_sheet)!!
```

Comment:
Please update the version number in config.gradle as well

Reply:
done

Reply:
I believe merge in master for now as we have a develop branch